github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml

```diff
@@ -42,7 +42,7 @@ periodics:
         requests:
           memory: "8Gi"
 - name: periodic-common-templates-nightly-centos-cdi-builder
-  cron: "30 3 * * *"
+  cron: "30 15 * * *"
```

Comment:
Same as above, please revert this value.

Reply:
Done